SharpMap/Rendering/Symbolizer/RasterPointSymbolizer.cs

```diff
@@ -104,6 +116,41 @@ internal ImageAttributes ImageAttributes
             }
         }
 
+        /// <summary>
+        /// Construct imageattribute based upon Transparency and/or Color Re-map
+        /// </summary>
+        private void ConstructImgAttributes()

```

Comment:
No abbreviation in function name `ConstructImageAttributes`

Reply:
Done.